app/src/main/res/layout/activity_profile.xml

```diff
@@ -6,174 +6,119 @@
     android:id="@+id/profile_layout"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@mipmap/sol_key"
     tools:layout_editor_absoluteY="50dp">
 
-<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    xmlns:tools="http://schemas.android.com/tools"
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:background="@mipmap/background1"
```

Comment:
i actually think this background color is better (its also the color i changed my menu activity to.
up to u but i think this should be our background.